ImageFilterEditor/ImageFilterEditor/Filtering/Models/ImageFilter.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ImageFilter.swift
+//  ImageFilterEditor
+//
+//  Created by Shawn Gee on 5/5/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+import UIKit
+
```

Comment:
This is really nice as well. I know you're not finished, but it would probably be a good idea to make the name and icon constants. And since the `filterControls` is a computed property using structs, the `ImageFilterLinearControl` is going to be essentially "reset" unless we save it elsewhere; maybe that's what you want, though? Definitely worth considering if you haven't yet, in terms of encapsulation & whatnot.